### PR TITLE
add realtime storj coin price

### PR DESCRIPTION
**Pull request by dylanlott**

### CODE REVIEW

File: lib/server/routes/coinpayments.js [hunk 1]
```diff
@@ -45,19 +65,34 @@ CoinpaymentsRouter.prototype.handleIPN = function (req, res) {
...
+      return this._currentPrice().then((price) => {
+        const data = {
+          proc,
+          current_price: price
+        }
+        return data;
+      });
+    })
+    .then((data) => {
+      console.log('formatted amount: ', formatted.amount);
```
Comment (bryanchriswhite) [inline lib/server/routes/coinpayments.js]: is this the same logger and/or is it worth logging?

File: lib/server/routes/coinpayments.js [hunk 2]
```diff
@@ -68,35 +103,35 @@ CoinpaymentsRouter.prototype.handleIPN = function (req, res) {
...
-              paid_amount: req.body.amount,
-              invoiced_amount: req.body.amount,
-              user: proc.user,
+              paid_amount: USD,
+              invoiced_amount: USD,
+              user: data.proc.user,
               paid: true,
               payment_processor: constants.PAYMENT_PROCESSORS.COINPAYMENTS,
-              type: constants.CREDIT_TYPES.AUTO,
+              type: constants.CREDIT_TYPES.MANUAL,
```
Comment (bryanchriswhite) [inline lib/server/routes/coinpayments.js]: why `MANUAL`?

File: lib/server/routes/coinpayments.js [hunk 3]
```diff
@@ -132,10 +167,10 @@ CoinpaymentsRouter.prototype._definitions = function () {
     ['POST', '/coinpayments',
       bodyParser.json(),
       bodyParser.urlencoded({ extended: true }),
-      Coinpayments.ipn({
-        'merchantId': process.env.CP_MERCHANT_ID,
-        'merchantSecret': process.env.CP_IPN_SECRET
-      }),
+      //Coinpayments.ipn({
```
Comment (bryanchriswhite) [inline lib/server/routes/coinpayments.js]: do we need to keep this?

**Comment by bryanchriswhite** [comment]
looks like you need to add `axios` to the package.json (from travis):
```
> storj-billing@0.2.0 testsuite /home/travis/build/Storj/billing
> STORJ_ALLOW_LOOPBACK=1 NODE_ENV=test ./node_modules/.bin/_mocha --recursive test/**
opening database connection at [object Object]
module.js:471
    throw err;
    ^
Error: Cannot find module 'axios'
    at Function.Module._resolveFilename (module.js:469:15)
    at Function.Module._load (module.js:417:25)
...
```

File: lib/server/routes/coinpayments.js (hunk 2 above)
Comment (dylanlott) [reply lib/server/routes/coinpayments.js]: Auto are ones that are subscription based, right? This isn't a subscription payment. They have to manually send the payments.

File: lib/server/routes/coinpayments.js (hunk 1 above)
Comment (dylanlott) [reply lib/server/routes/coinpayments.js]: Removed

File: lib/server/routes/coinpayments.js (hunk 3 above)
Comment (dylanlott) [reply lib/server/routes/coinpayments.js]: It's commented out for testing, uncommenting it for staging push.

**Pull request merged**